main/WebServer.cpp

```diff
@@ -496,6 +496,8 @@ namespace http {
 			RegisterCommandCode("downloadready", std::bind(&CWebServer::Cmd_DownloadReady, this, _1, _2, _3));
 			RegisterCommandCode("update_application", std::bind(&CWebServer::Cmd_UpdateApplication, this, _1, _2, _3));
 			RegisterCommandCode("deletedatapoint", std::bind(&CWebServer::Cmd_DeleteDatePoint, this, _1, _2, _3));
+			RegisterCommandCode("deletehistory", std::bind(&CWebServer::Cmd_DeleteHistory, this, _1, _2, _3));
```

Comment:
I would say avoid std::bind here.

Reply:
@neheb 
Thx for the review. Please understand I am not fluent in c++ 
I copied an existing function with similar functionality and only really changed the SQL's inside. 
So much obliged if you could elaborate a bit on why this std::bind is different then the ones already used and/or on the reason why this is bad coding practice. 

Always eager to learn!

Reply:
meh nvm. it's fine for this PR.